Tests/ArgumentParserUnitTests/HelpGenerationTests.swift

```diff
@@ -16,6 +16,21 @@ import ArgumentParserTestHelpers
 final class HelpGenerationTests: XCTestCase {
 }
 
+extension URL: ExpressibleByArgument {
```

Comment:
Nit: Can you indent with 2 spaces instead of 4?

Reply:
Oops, I fixed it now.